include/sys/zfs_refcount.h

```diff
@@ -76,6 +76,8 @@ int zfs_refcount_is_zero(zfs_refcount_t *);
 int64_t zfs_refcount_count(zfs_refcount_t *);
 int64_t zfs_refcount_add(zfs_refcount_t *, const void *);
 int64_t zfs_refcount_remove(zfs_refcount_t *, const void *);
+#define	zfs_refcount_remove_unless_last(rc, holder) 0 /* TODO */
```

Comment:
While it seems not required, I wonder if implementation would make CI coverage closer to real life.